src/client.ts

```diff
@@ -84,13 +84,27 @@ function unwrapData<T, X>(
   }
 }
 
+type ConstructorOptions = {
+  // The Plain API key
+  apiKey: string;
+
+  // Useful if you want to point to a different Plain environment
+  // as a Plain team-member
+  apiUrl?: string;
+
+  // Used to replace the global fetch implementation with a mock
+  // for testing. Should not be used in production.
+  __fetch?: typeof fetch;
```

Comment:
Why this approach and not `vi.spyOn(window, 'fetch', 'get').mockReturnValue(<mockFetch>)`

Reply:
Because we are in a node environment, and so there is no window!

Reply:
Good point!